### DISCUSSION TITLE
rootfs part size

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/tegra-demo-distro#79] [General]
Hello,
I would like to change the rootfs partition size for my Jetson nx xavier.
For this, I have added the following to the machine configuration:
ROOTFSPART_SIZE = "3000000000"

I see the partition table uses the APP_SIZE macro which is replaced by the ROOTFSPART_SIZE, so this should have worked fine.

I expected this to lead to ~2.5GB rootfs size but still I get the same, 4.3GB rootfs size.

My actual rootfs size is ~1.9GB (content wise and its RO).

Is there anything else I need to do?

**Comment by dwalkes** [comment]
HI @tirankas for mender root filesystem size customization see the suggestions at https://github.com/OE4T/tegra-demo-distro/wiki/Customization-of-Storage-Size-and-root-filesystem-size-on-Mender-Builds
